Helpers.ps1

```diff
@@ -384,7 +384,9 @@ function Write-Log
                 }
                 else
                 {
-                    $logFileMessage | Out-File -FilePath $Path -Append
+                    $stream = [System.IO.StreamWriter]::new($Path, [System.Text.Encoding]::UTF8)
+                    $stream.WriteLine($logFileMessage)
+                    $stream.Close()
```

Comment:
Because we need to ensure that this stream is closed, I think we need to do something more like this:

```suggestion
                    $stream = [System.IO.StreamWriter]::new($Path, [System.Text.Encoding]::UTF8)
                    try
                    {
                        $stream.WriteLine($logFileMessage)
                    }
                    finally
                    {
                        $stream.Close()
                        
```